persistence/pre_persistence/proto/block.proto

```diff
@@ -1,20 +1,22 @@
 syntax = "proto3";
+package pre_persistence;
+
+option go_package = "/persistence/pre_persistence/";
+
 import "google/protobuf/timestamp.proto";
-package utility;
-option go_package = "/utility/test/";
 
 message BlockHeader {
-  int64 Height = 1;
-  string Hash = 2;
-  string NetworkID = 3;
-  google.protobuf.Timestamp Time = 4;
-  uint32 NumTxs = 5;
-  int64 TotalTxs = 6;
-  string LastBlockHash = 7;
-  bytes ProposerAddress = 8;
+  int64 height = 1;
+  string hash = 2;
+  string network_id = 3;
+  google.protobuf.Timestamp time = 4;
+  uint32 num_txs = 5;
+  int64 total_txs = 6;
```

Comment:
_As a follow up to the convo above this. (_Could not suggest edits on the same thread_)_

Let's embrace the verbosity we are already benefiting from and name these fields as they should imply:

```suggestion
  uint32 block_txs_count = 5;
  int64 total_chain_txs_count = 6;
```